app/src/main/java/com/nutomic/syncthingandroid/service/SyncthingService.java

```diff
@@ -59,18 +59,8 @@
     public static final String ACTION_REFRESH_NETWORK_INFO =
             "com.nutomic.syncthingandroid.service.SyncthingService.REFRESH_NETWORK_INFO";
 
-    /**
-     * Callback for when the Syncthing web interface becomes first available after service start.
-     */
-    public interface OnWebGuiAvailableListener {
-        void onWebGuiAvailable();
-    }
-
-    private final HashSet<OnWebGuiAvailableListener> mOnWebGuiAvailableListeners =
-            new HashSet<>();
-
-    public interface OnApiChangeListener {
-        void onApiChange(State currentState);
+    public interface onServiceStateChangeListener {
```

Comment:
Interface should start with a capital letter. I am surprised lint does not bitch about it.